tests/unit/redirects/test_models.py

```diff
@@ -18,7 +17,9 @@ def get_factory(model):
     classes = [value for name, value in inspect.getmembers(factories, inspect.isclass)]
     all_factories = [c for c in classes if c.__name__.endswith("Factory")]
 
-    return first(all_factories, key=lambda m: m._meta.model == model)
+    return next(  # pragma: no branch
+        (f for f in all_factories if f._meta.model == model), None
+    )
```

Comment:
thought: Could this complain more loudly with `StopException`? Would we ever expect a model object doesn't have a factory?

Reply:
I've removed the defaults from all the tests.